15-additional_resources.Rmd

```diff
@@ -0,0 +1,17 @@
+# Additional resources {#AdditionalResources}
+
+There are a number of further resources to help with contributions to R.
+
+## R Contributor Site
+
+Contains information for people interested in contributing to R core, including details for the [R Contribution Working Group](https://contributor.r-project.org/working-group), the [R Contributors Slack workspace](https://contributor.r-project.org/slack) and upcoming [R Contributor events and meetups](https://contributor.r-project.org/events/). 
+
+## R Translation Teams
+
+There are a number of translation projects already underway or completed, more information on how to offer your help with one of these, contact the teams involved, and to find more detailed instructions for translating R Messages or R Manuals can be found on the [R Translation Teams site](https://translation.r-project.org/).
```

Comment:
Good spot, I'd used that as it was referenced in one of the issue comments, though looking at the two it seems like the original link I had included hadn't been updated since 2020 - switched over to the suggested site now